rest/functionsapitest/graphql_queries_test.go

```diff
@@ -284,18 +284,20 @@ func TestContextDeadline(t *testing.T) {
 		return
 	}
 	defer rt.Close()
+	timeout := 2 * time.Second
```

Comment:
Even 2 seconds seems like a long wait for a single unit test.  Is this length of time needed?